datahub-web-react/src/app/entity/shared/tabs/Dataset/Schema/SchemaTable.tsx

```diff
@@ -194,11 +193,8 @@ export default function SchemaTable({
                     columns={allColumns}
                     dataSource={rows}
                     rowKey="fieldPath"
-                    components={{
-                        body: {
-                            row: SchemaRow,
-                        },
-                    }}
```

Comment:
unfortunately removing this will also remove our functionality around showing foreign key entities on clicking them. So when you have a foreign key like this schema:
<img width="1226" alt="image" src="https://user-images.githubusercontent.com/28656603/198141777-10e6dd7b-fa84-4d01-8baa-a2c8068b21a6.png">
and you click on "Foreign Key" you get this:
<img width="1422" alt="image" src="https://user-images.githubusercontent.com/28656603/198141886-2e71cd17-d920-4d8a-8193-58a6a1371c9b.png">


Is there a way to use virtuallist while preserving this schema functionality to use `SchemaRow`? After playing around with it a bit myself it doesn't seem very easy. So we'd have to change our implementation to still get the same functionality that `SchemaRow` provides with foreign keys.

Another option is to look at other virtual list libraries that might still allow us to do this. Have you looked at [react-window](https://www.npmjs.com/package/react-window)? not sure how easy it would be to use with ant designs table, though.

Reply:
Hey Collin, first of all -- thank you for the thorough review! I'm currently in progress working on getting the foreign key information back to working with the chip toggle. 

It looks like you had originally started some work to have rows expand based on search (`filterSchemaRows.ts`), so I was going to build off of where that is used in the `expandable` prop for the table itself with `selectedFkFieldPath` info.

Originally I had tried to implement react-window with autosizer, but was running in to a wall due to how ant-design (really rc-table) expects to render the `components` overrides vs how react-window prefers to wrap virtualized rows.

I'm making good progress on restoring the FK functionality though and expect to have it pushed tomorrow afternoon after resolving some dependency issues.

Reply:
amazing! I appreciate the explanation - all that makes total sense to me